### PR TITLE
[Chore] CSS cleanup

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

File: static/style.css [hunk 1]
```diff
@@ -647,541 +647,6 @@ html {
...
-.CodeBlock-is-one-liner > code,
-.CodeBlock-scrolls-horizontally > code {
-  word-break: normal;
-  overflow-x: auto;
-  -webkit-overflow-scrolling: touch;
-}
-
-.CodeBlock-scrolls-horizontally > code {
-  white-space: pre;
-}
```
Comment (kodster28) [inline static/style.css:742]: Bottom to here is 1-to-1 in codeBlock.css 1:93.

File: static/style.css [hunk 2]
```diff
@@ -647,541 +647,6 @@ html {
...
-
-.CodeBlock--token-entity {
-  color: var(--code-blue);
-  cursor: help;
-}
-
-.CodeBlock--token-inserted,
-.CodeBlock--token-inserted-sign {
-  color: var(--code-green);
-}
```
Comment (kodster28) [inline static/style.css:1013]: Through here, codeBlock.css 202-364.

File: static/style.css [hunk 3]
```diff
@@ -647,541 +647,6 @@ html {
...
-  + .CodeBlock--row-content
-  > .CodeBlock--token-plain:empty
-  + .CodeBlock--token-doc-comment.CodeBlock--token-comment:empty:after,
-.CodeBlock--row
-  > .CodeBlock--row-indicator:empty
-  + .CodeBlock--row-content
-  > .CodeBlock--token-table.CodeBlock--token-table-data-rows:empty
-  + .CodeBlock--token-plain:empty:after {
-  content: " ";
-}
```
Comment (kodster28) [inline static/style.css:1183]: Code block 366-534

File: static/style.css [hunk 4]
```diff
@@ -647,541 +647,6 @@ html {
...
-.CodeBlock-with-rows .CodeBlock--row {
-  position: relative;
-  display: block;
-  width: 100%;
-}
-
-.CodeBlock-with-rows .CodeBlock--row-content {
-  display: block;
-  white-space: pre-wrap;
-  padding: 0 var(--padding-horizontal);
```
Comment (kodster28) [inline static/style.css:834]: This will technically differ if we merge in #14028 first.

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**